app/pages/solutions-list/controller.js

```diff
@@ -12,9 +12,28 @@ const getSolutionListData = async (filterType) => {
   throw new Error(`No endpoint found for filter type: ${filterType}`);
 };
 
+const transformCapabilities = ({ capabilitiesSelected }) => {
```

Comment:
worth moving to it's own file?

Reply:
I don't think so

Reply:
😂 only reason I say that is there is nothing that currently test this transformation

Reply:
there are no tests for this whole controller ...